app/views/state_file/questions/eligible/edit.html.erb

```diff
@@ -34,6 +34,10 @@
       <div class="with-top-separator">
         <p class="text--bold"><%=t('.want_to_claim_heading') %></p>
         <p><%=t('.want_to_claim_vita', state_name: state_name) %></p>
+        <p>
+          <% vita_link = params[:us_state] == "az" ? "https://airtable.com/appnKuyQXMMCPSvVw/pag0hcyC6juDxamHo/form" : "https://airtable.com/appQS3abRZGjT8wII/pagtpLaX0wokBqnuA/form" %>
```

Comment:
It seems like we are duplicating these links in the codebase.  Might be good to make them a constant somewhere that can just be referenced everywhere.